SharperUniverse/Persistence/PersistenceManager.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System;
+using System.Collections.Generic;
+using SharperUniverse.Core;
+
+namespace SharperUniverse.Persistence
+{
+	public class PersistenceManager
+	{
+		internal IDatabaseProvider _provider;
+
+		public PersistenceManager(Type provider, string connectionString)
+		{
+			_provider = (IDatabaseProvider)Activator.CreateInstance(provider);
+			_provider.ConnectionString = connectionString;
+		}
+
+		public int Save()
+		{
+			foreach(var component)
+		}
+
+		public List<BaseSharperComponent> Load(int saveIndex)
```

Comment:
Perhaps consider making this a generic method that can return a `List<T>` where `T` is the specified component?